lib/commands/-command.js

```diff
@@ -6,7 +6,7 @@ module.exports = Command.extend({
   init() {
     process.env.EMBER_CLI_ELECTRON = true;
 
-    if (this._super && this._super.init) {
+    if (this._super && this._super.init && this._super.init !== this.init) {
```

Comment:
Plausible -- I don't remember exactly how, but I'm pretty sure in older version of ember-cli invoking super-class functions was more fiddly.